public/_headers

```diff
@@ -5,7 +5,7 @@
   Content-Security-Policy: object-src 'none'; 
   Content-Security-Policy: base-uri 'self'; 
   Content-Security-Policy: default-src 'self; 
-  Content-Security-Policy: script-src 'self' https://*.lunie.io; 
+  Content-Security-Policy: script-src 'self' https://*.lunie.io https://deploy-preview-*--lunieio.netlify.app; 
```

Comment:
```suggestion
  Content-Security-Policy: script-src 'self' https://*.lunie.io https://*--lunieio.netlify.app; 
```

Reply:
yeah, that wasn't working 👍 

Now that seems fixed, but the problem is that I'm getting a `Note that 'img-src' was not explicitly set, so 'default-src' is used as a fallback` when the `img-src` is already set and, in fact, even before the actual `default-src`.

Really weird.

Will continue looking into this tomorrow. Cannot explain why this happens